installer/conf/container.conf

```diff
@@ -53,26 +53,50 @@
 # Container service log
 <source>
 	type omi
-	run_interval 30s
+	run_interval 15s
 	tag oms.container.log
 	items [
 		["root/cimv2","Container_ContainerLog"]
 	]
 </source>
 
+# Container host inventory
+<source>
+	type omi
+	run_interval 60s
+	tag oms.api.ContainerNodeInventory
+	items [
+		["root/cimv2","Container_HostInventory"]
+	]
+</source>
+
 # Filter for correct format to endpoint
 <filter oms.container.**>
 	type filter_container
 </filter>
 
+<match oms.api.ContainerNodeInventory**>
+  type out_oms_api
+  log_level debug
+  buffer_chunk_limit 8m
+  buffer_type file
+  buffer_path %STATE_DIR_WS%/out_oms_containerinventory*.buffer
+  buffer_queue_limit 1024
+  flush_interval 20s
+  retry_limit 10
+  retry_wait 15s
+  max_retry_wait 9m
+</match>
+
+
 #separate buffer for containers
 <match oms.container.**>
   type out_oms
   log_level info
-  buffer_chunk_limit 20m
+  buffer_chunk_limit 8m
   buffer_type file
   buffer_path %STATE_DIR_WS%/out_oms_docker*.buffer
-  buffer_queue_limit 20
+  buffer_queue_limit 1024
```

Comment:
Pretty big jump (noticed same buffer_queue_limit up above). That seems pretty big, is that really what you want? How would this impact disk space on customer systems? Is this a situation where the customer system may not ever catch up from a backlog that's that large?